.travis.yml

```diff
@@ -2,6 +2,6 @@ sudo: required
 dist: trusty
 language: node_js
 node_js:
-  - "stable"
+  - 17
```

Comment:
Instead of specifying node version in `.travis.yml`, (which only travis-ci reads), can we specify it in `.nvmrc` so that developers can be sure they're on the right version (via `nvm use`). (Travis-ci will also respect what's in `.nvmrc`.) Also, we should prob specify v 14 unless we know that we can safely update any app that uses this wrapper to Node 17.